Automattic/woocommerce-payments
# Trigger the Jetpack connection flow from the plugin UI

Integrates the Jetpack connection flow with the existing WCPay onboarding flow. The complete flow would be like this:
- Click the "Setup" button on the Connect WCPay page.
- If there's no Jetpack connection, redirect to wp.com to the Jetpack Connect flow.
- When the Jetpack flow ends, if WCPay is not setup, automatically redirect to Stripe's KYC flow.
- Once that flow ends, the merchant is dropped into the WCPay settings screen.

Alternate flows:
- If the site is already connected to Jetpack, clicking `Setup` will start the Stripe KYC flow.
- If the site was *not* connected to Jetpack, but when the Jetpack connection flow is completed it turns out that the site already has a WCPay account configured, the user will be dropped into the WCPay Settings page instead of starting the Stripe KYC flow. You can test this by having WCPay fully configured and then disconnecting the site from Jetpack.
- If the flow was initiated from the WC-Admin setup task list, the user will be returned to there (instead of WCPay Settings). Note: I haven't tested this yet.

To test:
- Pick the main flow or one of the alternate flows and try to get to a fully functioning WCPay extension.

Note that you still need Jetpack-the-plugin installed and activated to test this.

Also note that the Jetpack Connect design still follows the Jetpack color scheme, there will be several PRs in WP-Calypso to change that.
